### PR TITLE
Update footer

### PR DESCRIPTION
**Pull request by borislavpetrovhmcts**
Footer update

Fixes issue EUI-214

### CODE REVIEW

File: src/app/containers/footer/footer.component.spec.ts [hunk 1]
```diff
@@ -15,6 +15,8 @@ describe('FooterComponent', () => {
 
     let testHostComponent: TestDummyHostComponent;
     let testHostFixture: ComponentFixture<TestDummyHostComponent>;
+    let el: DebugElement;
```
Comment (andywilkinshmcts) [inline src/app/containers/footer/footer.component.spec.ts]: these variables don't appear to be used

File: src/app/components/hmcts-global-footer/hmcts-global-footer.component.spec.ts [hunk 2]
```diff
@@ -19,20 +18,40 @@ describe('HmctsGlobalFooterComponent', () => {
         @ViewChild(HmctsGlobalFooterComponent)
         public hmctsGlobalFooterComponent: HmctsGlobalFooterComponent;
     }
-
+    let testHostComponent: TestDummyHostComponent;
+    let testHostFixture: ComponentFixture<TestDummyHostComponent>;
+    let el: DebugElement;
+    let de: any;
```
Comment (andywilkinshmcts) [inline src/app/components/hmcts-global-footer/hmcts-global-footer.component.spec.ts]: These variables don't appear to be used

File: src/app/containers/footer/footer.component.spec.ts (hunk 1 above)
Comment (borislavpetrovhmcts) [reply src/app/containers/footer/footer.component.spec.ts]: They are used. testHostComponent in line 36, testHostFixture - line 35 and 36, variable el is now removed

File: src/app/components/hmcts-global-footer/hmcts-global-footer.component.spec.ts (hunk 2 above)
Comment (borislavpetrovhmcts) [reply src/app/components/hmcts-global-footer/hmcts-global-footer.component.spec.ts]: These are now removed

File: src/app/app.constants.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const FooterData =  {
+  heading: 'Help',
```
Comment (andywilkinshmcts) [inline src/app/app.constants.ts]: Is there a reason why this isn't in the config.json, and exposed through the AppConfigService? 

It's OK for now, but please add a backlog task to refactor this into the config service.

Comment (borislavpetrovhmcts) [reply src/app/app.constants.ts]: This is static content which never changes. This is why it is in constants file.

Comment (fazanki) [reply src/app/app.constants.ts]: i agree with Boris - don't think this needs to be configurable, so no need to be in config file unless is decided to be configurable by the business

**Pull request merged**